op-program/client/l2/engineapi/l2_engine_api.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/ethereum/go-ethereum/eth/downloader"
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/params"
+	"github.com/holiman/uint256"
```

Comment:
import lint issue?

Reply:
because of grouping? We don't enforce a specific import grouping scheme (yet)

Reply:
fixed all imports